File: Snooper/src/test/java/com/prateekj/snooper/networksnooper/activity/HttpCallTabTest.java
package com.prateekj.snooper.networksnooper.activity;

import com.prateekj.snooper.R;

import org.junit.Test;

import static com.prateekj.snooper.networksnooper.activity.HttpCallTab.HEADERS;
import static com.prateekj.snooper.networksnooper.activity.HttpCallTab.REQUEST;
import static com.prateekj.snooper.networksnooper.activity.HttpCallTab.RESPONSE;
import static org.hamcrest.CoreMatchers.is;
import static org.junit.Assert.assertThat;

public class HttpCallTabTest {
  @Test
  public void shouldReturnResponseTabDetails() throws Exception {
    assertThat(RESPONSE.getTabTitle(), is(R.string.response));
  }

  @Test
  public void shouldReturnRequestTabDetails() throws Exception {
    assertThat(REQUEST.getTabTitle(), is(R.string.request));
  }

  @Test
  public void shouldReturnHeadersTabDetails() throws Exception {
    assertThat(HEADERS.getTabTitle(), is(R.string.headers));
  }
}

File: Snooper/src/test/java/com/prateekj/snooper/networksnooper/activity/HttpCallTabTest.kt
package com.prateekj.snooper.networksnooper.activity

import com.prateekj.snooper.R

import org.junit.Test

import com.prateekj.snooper.networksnooper.activity.HttpCallTab.HEADERS
import com.prateekj.snooper.networksnooper.activity.HttpCallTab.REQUEST
import com.prateekj.snooper.networksnooper.activity.HttpCallTab.RESPONSE
import org.hamcrest.CoreMatchers.`is`
import org.junit.Assert.assertThat

class HttpCallTabTest {
    @Test
    fun shouldReturnResponseTabDetails() {
        assertThat(RESPONSE.tabTitle, `is`(R.string.response))
    }

    @Test
    fun shouldReturnRequestTabDetails() {
        assertThat(REQUEST.tabTitle, `is`(R.string.request))
    }

    @Test
    fun shouldReturnHeadersTabDetails() {
        assertThat(HEADERS.tabTitle, `is`(R.string.headers))
    }
}